l10n_br_sale_product/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extends the standard Odoo sale module for Brazil. Especially it deals with the propagation of the fiscal operation and fiscal comment. This module can be used both with or without the "NFe" electronic invoicing so it does not provide the NFe support itself.
```

Comment:
sorry @mbcosta I made a mistake: l10n_br_sale_product is specially designed for sales with NFe. So please chane it to just:

This module extends the standard Odoo sale module for Brazil, specially when selling products with NFe's.

It deals with the propagation of the fiscal operation and fiscal comment. I also displays the following sale total details:

* untaxed amount
* taxes
* gross amount (valor bruto)
* shipping (frete)
* costs (outros custos)
* insurance (seguro)